src/_locales/en/messages.json

```diff
@@ -113,6 +113,14 @@
     "message": "Long Break",
     "description": "Long Break section header. Shown on the settings page."
   },
+  "startup_title": {
```

Comment:
Internally, I'd prefer `autostart` over `startup` when referring to this feature. Ultimately, this might be called "Automatic Start" or "Autostart" in the UI.